index.html

```diff
@@ -4722,11 +4738,163 @@ <h2>Basic Concept</h2>
         The next update of the draft will also introduce an import mechanism that allows to import other Thing Models and to reuse sub-definitions
         of such models.
     </p>
+</section>
+
+
+<section id="thing-model-td-placeholder" class="normative">
+    <h3>Placeholder</h3>
+
+<p>
+    <a>Thing Model</a> can specify which terms should be taken in a TD instance, but their values are unspecific and are first known during TD instantiation. 
+    In such a case the placeholder labeling MAY be used in Thing Model that MUST be substituted with a concrete value when TD instance is created 
+    from the Thing Model. 
+    <span class="rfc2119-assertion" id="td-model-placeholder">
+        The string-based pattern of the placeholder MUST follow <code>{&ZeroWidthSpace;{PLACEHOLDER_IDENTIFIER}}</code>, 
```

Comment:
I don't think "&ZeroWidthSpace;" is needed

Reply:
strange I did not add this